### ISSUE TITLE
How to ensemble different models, like Resnet50 and Resnet152?

### ISSUE THREAD

**Issue by cuge1995** [TorchEnsemble-Community/Ensemble-Pytorch#83]

**Comment by xuyxu** [comment]
Hi @cuge1995, the heterogeneous ensemble is not supported so far since training configurations of different base estimators could be quite different (e.g., different type of optimizer, different learning rate, ...). It is hard to find a user-friendly way to specify the training configuration for each base estimator in the ensemble.
